### PR TITLE
Add missing dispose calls to fix program hangs on Linux

### PR DESCRIPTION
**Pull request by jasonleenaylor**
* Dispose of temporary bitmaps as well as the Disabled grey
  scale bitmaps in BitmapButton
* Dispose of SettingsProtectionHelper components and timer

### CODE REVIEW

File: PalasoUIWindowsForms/SettingProtection/SettingsProtectionHelper.cs [hunk 1]
```diff
@@ -154,6 +154,9 @@ public ISite Site
 		private bool _isDisposed = false;
 		public void Dispose()
 		{
+			base.Dispose();
```
Comment (papeh) [inline PalasoUIWindowsForms/SettingProtection/SettingsProtectionHelper.cs]: good catch.  Should this method not be `override`?

File: PalasoUIWindowsForms/Widgets/BitmapButton.cs [hunk 2]
```diff
@@ -746,31 +748,33 @@ private void paint_ImageBorder(System.Drawing.Graphics g, System.Drawing.Rectang
...
+					//
+					// draw line 1
+					//
+					border_Contract(1, ref pts);
+					g.DrawLines(pen0, pts);
+					//
+					// release resources
+					//
+					pen1.Dispose();
+					pen0.Dispose();
```
Comment (papeh) [inline PalasoUIWindowsForms/Widgets/BitmapButton.cs]: `using(pen0)`
`using(pen1)`

File: PalasoUIWindowsForms/SettingProtection/SettingsProtectionHelper.cs (hunk 1 above)
Comment (jasonleenaylor) [inline PalasoUIWindowsForms/SettingProtection/SettingsProtectionHelper.cs]: Actually, there should never have been a Dispose() here in the first place. Thanks for catching this, apparently I was trying to rush my change in before the weekend.

File: PalasoUIWindowsForms/Widgets/BitmapButton.cs (hunk 2 above)
Comment (jasonleenaylor) [inline PalasoUIWindowsForms/Widgets/BitmapButton.cs]: done

**Comment by jasonleenaylor** [comment]
Windows test failure seems unrelated. Looks like a fragile test.

**Pull request merged**